src/client/pages/Offer/Introduction/HeroOfferDetails.tsx

```diff
@@ -121,7 +121,7 @@ export const HeroOfferDetails: React.FC<Props> = ({
     return query.get('showEdit') === 'true'
   })
   const { person } = offerData
-  const numberCoInsured = person.householdSize - 1
+  const numberCoInsured = (person.householdSize ?? 0) - 1
```

Comment:
Wait... if we display this, won't it show up as `-1` co-insured now?

Reply:
Yeah, it's not easy to spot, but on row 136 we do a conditional render if `numberCoInsured > 0`

Reply:
Would still consider changing to:

```suggestion
  const numberCoInsured = (person.householdSize ?? 1) - 1
```

...or does that not work? Just since "-1" co-insured or household size "0" simply is not valid.